README.md

```diff
@@ -15,18 +15,28 @@ The `docker-compose` file will both
 1. setup the postgres instance
 2. start the graphql engine
 
+First you will need to build the docker images, and pass the build argument to the `docker-compose build` command.
+
+```
+docker-compose build --build-arg FRONTEND_DIRECTORY="<location of frontend directory>"
+```
+
+You can clone the frontend from [here](https://github.com/gs0510/current-bench/).
```

Comment:
shouldn't this point to craigfe/current-bench?

Reply:
Oh yes!